Doc/library/datetime.rst

```diff
@@ -719,13 +719,25 @@ Instance methods:
    .. versionchanged:: 3.9
       Result changed from a tuple to a :term:`named tuple`.
 
-.. method:: date.isoformat()
 
```

Comment:
I think you added a line here. I'll delete this suggestion when it has been applied, to hide it from the files tab.

```suggestion
```

Reply:
Yes, because all the other methods are separated by two lines in this file. Consistency here is more important (at least, that's what I argue as a Sphinx maintainer).

I only apply such cosmetic changes if I am modifying the corresponding lines. For instance, I would not normalize the style of a file on an untouched function even though it would make it consistent (at least not a C or a Python file, docs are a bit different since their primary target is clarity).

Reply:
`git` isn't great at detecting stylistic changes: it doesn't ignore `else` and new lines resulting in very noisy diffs.
That's why I usually try to keep the diff minimal even if I'm modifying a function.
(that can make the difference between the diff being rendered by default or not here on GitHub)